site/src/components/jurisdiction-info/Details.tsx

```diff
@@ -8,11 +8,16 @@ import WorkReqs from "./WorkReqs";
 const Details = ( j: Jurisdiction ) => (
    <div>
       <h2>{j.name}, {j.state.alpha}</h2>
-      <p>{j.jurisdiction_link_text}</p>
+      { j.jurisdiction_link_text && j.jurisdiction_link &&
+         (<p>
+            {j.jurisdiction_link_text}
+            {" "}
+            <a href={`/jurisdiction/${j.jurisdiction_link.id}`} >click here</a>.
```

Comment:
I think an anchor will reload the page, so probably want to use `stencil-route-link`